examples/client/trusted-types-overlay/README.md

```diff
@@ -0,0 +1,36 @@
+# client.overlay.policyName option
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  // ...
+  output: {
+    trustedTypes: { policyName: "webpack" },
+  },
+  devServer: {
+    client: {
+      overlay: {
+        policyName: "overlay-policy",
```

Comment:
Consider using `webpack#dev-overlay` or something like this to feature the recommended naming scheme.

Reply:
Fixed in https://github.com/webpack/webpack-dev-server/pull/4404/commits/8ef1a488fff474ff39097fc1394f6486b52b97b1